2-mandatory/2-retrying-random-numbers.js

```diff
@@ -11,6 +11,11 @@ function generateRandomNumber() {
 
 function getRandomNumberGreaterThan50() {
     // TODO - implement using a do-while loop
+do{
+    let number = generateRandomNumber();
+}
+while (number <= 50);
+return number;
 }
 
 /* ======= TESTS - DO NOT MODIFY ===== */
```

Comment:
Did you test this one? When I test this code I get the error "ReferenceError: number is not defined" in Line 17, do you know why this is?